lib/stringToChars.js

```diff
@@ -0,0 +1,13 @@
+
+
+function stringToChars(string){
+    let arrayChars = [];
+    for (let i of string){
```

Comment:
You can also do `string.split('')`. The `.split()` function would split `cat` and return the array `['c', 't']`, but if you pass `''` (empty string) as argument, it splits by empty strings, which is on the left/right of every letter, so you get an array of all the characters other than the empty string, which is `['c', 'a', 't']`